superset/assets/src/CRUD/crud.less

```diff
@@ -24,24 +26,24 @@
     padding: 10px;
   }
   .control-label {
-    font-weight: bold;
-    font-size: 16px;
+    font-weight: @font-weight-bold;
+    font-size: @font-size-l;
```

Comment:
let's not use abbreviations here, maybe call it `font-size-large`?

Reply:
Thought it might be better to be systematic, since the sizes range is `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`